test/unit-test/KuksavalUnitTest.cpp

```diff
@@ -233,21 +230,21 @@ BOOST_AUTO_TEST_CASE(Test_GetMetaDataService){
             \"Vehicle\": {\
                 \"children\": {\
                     \"Speed\": {\
-                        \"datatype\": \"int32\",\ 
-                        \"description\": \"Vehicle speed, as sensed by the gearbox.\",\ 
-                        \"max\": 250,\ 
-                        \"min\": -250,\ 
-                        \"type\": \"sensor\",\ 
-                        \"unit\": \"km/h\",\ 
+                        \"datatype\": \"int32\",\
```

Comment:
```suggestion
                        \"datatype\": \"int32\",
```

But why rm all ending `\`

Reply:
There were many warnings from the compiler, because there was a space after \